lib/client.js

```diff
@@ -1,23 +1,25 @@
-exports.trace = (...message) => {
-  console.trace(...message)
+'use strict'
+
+exports.trace = function () {
+  console.trace(Array.prototype.slice.call(arguments).toString())
 }
 
-exports.debug = (...message) => {
-  console.log(...message)
+exports.debug = function () {
+  console.log(Array.prototype.slice.call(arguments).toString())
 }
 
-exports.info = (...message) => {
-  console.info(...message)
```

Comment:
you could replace this with ```console.error.apply(console, arguments)``` and it would behave more appropriately

Reply:
dig it!